tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    enum { buffer_size = 16 };
```

Comment:
hmmm, don't know if I like this approach.
I won't object, but at least for me its uncommon to initialize arrays with an `enum` value.